server/unithand.cpp

```diff
@@ -3592,7 +3606,8 @@ static void see_combat(struct unit *pattacker, struct unit *pdefender)
    Send combat info to players.
  */
 static void send_combat(struct unit *pattacker, struct unit *pdefender,
-                        int att_veteran, int def_veteran, int bombard)
```

Comment:
You've removed `int bombard` but #2362 needs it

Reply:
This seems to be the only compatibility issue between the two PRs - as far as I can tell

Reply:
Oh, I see that #2362 uses this to not route bombardment towards the combat HUD. We may need to have a discussion about the client-side display of bombardment before we merge.